clang/lib/Driver/Driver.cpp

```diff
@@ -880,6 +880,12 @@ void Driver::CreateOffloadingDeviceToolChains(Compilation &C,
   if (SYCLTargets && SYCLfpga)
     Diag(clang::diag::err_drv_option_conflict)
         << SYCLTargets->getSpelling() << SYCLfpga->getSpelling();
+ // -ffreestanding cannot be used with -fsycl
+  if (C.getInputArgs().hasArg(options::OPT_fsycl) &&
+      C.getInputArgs().hasArg(options::OPT_ffreestanding)) {
+    Diag(clang::diag::err_drv_option_conflict) << "-fsycl"
+                                               << "-ffreestanding";
```

Comment:
It would be preferable to use `Diag(clang::diag::err_drv_expecting_fsycl_with_sycl_opt)` and the `getSpelling()` method for the FFreeStanding option as in the above blocks.

Reply:
As FFreeStanding is not a pointer anymore, I am not able to use getSpelling() here.